elliott

```diff
@@ -748,7 +767,47 @@ Example to look for bugs that have been moved back to MODIFIED
                           comment = "Elliott changed bug status from {initial} to {final}.".format(initial=original_state, final=new_state),
                           private = True)
     
-    green_print("{count} bugs successfullly modified".format(bug_count))
+    green_print("{count} bugs successfullly modified".format(changed_bug_count))
+
+
+#
+# Search for CVE tracker bugs
+#
+@cli.command("find-cve-trackers", short_help="Find and list CVE tracker bugs forSecurity issues.")
+@click.option('--cve',
+              required=False,
+              help="CVE number to filter on. (ex. CVE-2011-1000000)")
+@click.option('--status', 'status',
+              multiple=True,
+              required=False,
+              default=['NEW', 'ASSIGNED', 'POST', 'MODIFIED', 'ON_QA', 'VERIFIED', 'RELEASE_PENDING'],
+              type=click.Choice(elliottlib.constants.VALID_BUG_STATES),
+              help="Status the bugs can be in.")
+@pass_runtime
+def find_cve_trackers(runtime, cve, status):
+    """Find Red Hat Bugzilla security bugs and list them out. Automatic attachment of these
+    bugs is not supported because security issues generally need to be hand crafted to make
+    sure all requirements are met.
+
+    Usage:
+\b
+    $ elliott --group openshift-3.7 find-cve-trackers
```

Comment:
What does that example do? Are there several other common ways one might run this?